python/dataAPIutilities.py

```diff
@@ -155,31 +161,42 @@ def getInstantiatingClass(api, functionObject):
     # Andrew please add exactly what we want to the Confluence page
```

Comment:
Update comment with issue# here

Reply:
This is already there (including a detailed example), but you haven't given it an issue number (because it isn't a "bug fix" but a "feature request").

Do you want me to add a bug number to the Confluence page?